contracts/AaveLenderBorrowerCloner.sol

```diff
@@ -0,0 +1,92 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity 0.6.12;
+pragma experimental ABIEncoderV2;
+
+import "./Strategy.sol";
+
+contract AaveLenderBorrowerCloner {
+    address public immutable original;
+
+    event Cloned(address indexed clone);
+    event Deployed(address indexed original);
+
+    constructor(
+        address _vault,
+        address _yVault,
+        bool _isWantIncentivised,
+        bool _isInvestmentTokenIncentivised,
+        string memory _strategyName
+    ) public {
+        Strategy _original = new Strategy(_vault, _yVault, _strategyName);
```

Comment:
I would do `original = xxx` directly.

Reply:
not important but it saves gas to use a local variable vs a state variable